package.json

```diff
@@ -0,0 +1,27 @@
+{
+  "name": "dav-client",
+  "version": "0.0.1",
+  "description": "A DAV client to connect to DAV servers",
+  "main": "build/main.js",
+  "scripts": {
+    "build": "rollup -c",
+    "dev": "rollup -c -w"
+  },
+  "devDependencies": {
+    "@rollup/plugin-commonjs": "^17.1.0",
+    "@rollup/plugin-node-resolve": "^11.2.0",
+    "@rollup/plugin-typescript": "^8.2.0",
+    "@tsconfig/recommended": "^1.0.1",
+    "@types/url-join": "^4.0.0",
+    "rollup": "^2.42.0",
+    "rollup-plugin-terser": "^7.0.2",
+    "tslib": "^2.1.0",
+    "typescript": "^4.2.3"
+  },
```

Comment:
We need some testing library too :wink: (jest)

Reply:
I'll add Jest when I start writing tests for these pieces of code (which is after you guys have finished reviewing the code :smile:).

Reply:
Jest & tests are added now.